src/pybind/mgr/dashboard_v2/HACKING.rst

```diff
@@ -505,3 +505,22 @@ The unit test code will look like the following::
           rbd_list_mock.return_value = ['img1', 'img2']
           self._get('/api/rbdimages')
           self.assertJsonBody([{'name': 'img1'}, {'name': 'img2'}])
+
+
+How to execute unit tests without a running Ceph cluster?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are some unit tests that don't require a running Ceph cluster. Executing
+these tests can be done manually by running ``py.test``::
+
+  $ . .tox/py3/bin/activate  # for Python 2.7: . .tox/py27/bin/activate
```

Comment:
Maybe we should mention here how the .tox directory is created, because it does not exist until tox is executed once.